app/controllers/submit.js

```diff
@@ -284,6 +278,27 @@ export default Controller.extend(Analytics, BasicsValidations, NodeActionsMixin,
 
     moderationType: alias('currentProvider.reviewsWorkflow'),
 
+    supplementalChanged: computed('supplementalProjectTitle', 'pendingSupplementalProjectTitle', 'selectedSupplementalProject', 'node', function() {
```

Comment:
Lots of logic here because unlike most of the form sections, supplemental materials behaves differently on "Submit" than "Edit". On submit, changes are merely staged, and made to appear as if they've been saved, when they're actually not persisted to the server until the preprint is finalized. This is to prevent a bunch of projects being created for the user inadvertently. On edit, supplemental material changes are saved immediately, per the general logic of the form.